### ISSUE TITLE
Vertical filtering on inserts

### ISSUE THREAD

**Issue by rvernica** [begriffs/postgrest-docs#130]
For insertions, the [docs](https://postgrest.com/en/v4.4/api.html#insertions-updates) talk about using `Prefer: return=representation` to get the inserted object back. Moreover they also talk about applying [vertical filtering](https://postgrest.com/en/v4.4/api.html#v-filter) to it, e.g., `?select=foo`.

Where would one specify the `select=foo` part in the `POST` request of an insert operation?

**Comment by steve-chavez** [comment]
Yeah, that's a bug reported in https://github.com/begriffs/postgrest/issues/1087, this will be fixed on the next release.
